### ISSUE TITLE
Behavior properties disapear when required behavior is named Behavior

### ISSUE THREAD

**Issue by arthuro555** [4ian/GDevelop#3031]
## Describe the bug

When adding to a custom behavior a required behavior and giving the property the name `Behavior`, the other properties from the behavior are not visible in the event sheet anymore.

## To Reproduce

![npsk09jIIE](https://user-images.githubusercontent.com/19349038/132962734-ad949940-3e12-448a-b6a5-ff2a3e3f1705.gif)

## Other details
* GDevelop beta 117 on Windows 10

**Comment by 4ian** [comment]
Good catch, we should ideally prevent to use existing parameter names - at least we can hardcode that names like `Object` and `Behavior` are not allowed.
